packages/website/plugins/generated-rule-docs.ts

```diff
@@ -193,35 +194,79 @@ export const generatedRuleDocs: Plugin = () => {
         type: 'paragraph',
       } as mdast.Paragraph);
 
+      // need this, so we don't have comments inside the JSON objects
+      const getEslintrcStr = (withComment: boolean): string => {
+        return `{
+  "rules": {${
+    withComment
+      ? '\n    // Note: you must disable the base rule as it can report incorrect errors'
+      : ''
+  }
+    "${extendsBaseRuleName}": "off",
+    "@typescript-eslint/${file.stem}": "${optionLevel}"
+  }
+}`;
+      };
+
       root.children.splice(howToUseH2Index + 1, 0, {
         lang: 'js',
         type: 'code',
         meta: 'title=".eslintrc.cjs"',
-        value: `module.exports = {
-  "rules": {
-    // Note: you must disable the base rule as it can report incorrect errors
-    "${extendsBaseRuleName}": "off",
-    "@typescript-eslint/${file.stem}": "${optionLevel}"
-  }
-};`,
+        value: `module.exports = ${getEslintrcStr(true)};`,
       } as mdast.Code);
+
+      root.children.splice(howToUseH2Index + 2, 0, {
```

Comment:
You know, I think the link would be just a _little_ better if it was given `float: right`. That way it doesn't take up any extra vertical space:

<img width="896" alt="Screenshot of a right-aligned 'Click here to try this rule in the playground' just under its code block" src="https://user-images.githubusercontent.com/3335181/213903736-1eedced9-3039-4eb2-ad8b-bb5a4fc1d868.png">

The best way to do this would be to add a new component to `packages/website/src/theme/MDXComponents/index.tsx`. You can see how the `<rule-attributes>` component is used in code for a reference.

Is this something you have interest & time for in this PR? No worries if not (or you think it's better this way!), we can always file a followup issue.